From 85c7a0b1047056ca4a14429476a365ec1574f6a8 Mon Sep 17 00:00:00 2001
From: Tung Huynh <tung75605@gmail.com>
Date: Fri, 15 Dec 2023 15:38:43 -0800
Subject: [PATCH 9/9] gnutls: dont apply UNICODE patch

for some reason this patch breaks win32 compilation in new mingw-llvm versions
---
 contrib/src/gnutls/rules.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/src/gnutls/rules.mak b/contrib/src/gnutls/rules.mak
index 2ac81dc5e1..4450e7d8f7 100644
--- a/contrib/src/gnutls/rules.mak
+++ b/contrib/src/gnutls/rules.mak
@@ -22,7 +22,7 @@ gnutls: gnutls-$(GNUTLS_VERSION).tar.xz .sum-gnutls
 	$(APPLY) $(SRC)/gnutls/gnutls-fix-mangling.patch
 
 	# backport gnulib patch
-	$(APPLY) $(SRC)/gnutls/0001-Don-t-assume-that-UNICODE-is-not-defined.patch
+	# $(APPLY) $(SRC)/gnutls/0001-Don-t-assume-that-UNICODE-is-not-defined.patch
 
 	# fix forbidden UWP call which can't be upstreamed as they won't
 	# differentiate for winstore, only _WIN32_WINNT
-- 
2.42.0.windows.2

